src/core/textures/Texture.js

```diff
@@ -143,6 +144,13 @@ export default class Texture extends EventEmitter
             baseTexture.once('loaded', this.onBaseTextureLoaded, this);
         }
 
+        /**
+         * Anchor point that is used as default if sprite is created with this texture
+         * @type {PIXI.Point}
+         * @default {0,0}
+         */
+        this.anchor = anchor ? new Point(anchor.x, anchor.y) : new Point(0, 0);
```

Comment:
I think this `anchor` property name is problematic for reasons that @themoonrat mentioned. Another idea might be `inheritedAnchor`, `defaultAnchor` or something else? Another approach is that we mark this as `@private` doc since it's only used now internal to support TexturePacker's new feature and don't want to create confusion for users or expose new APIs.

Reply:
The feature is good even without TexturePacker. Setting texture default anchor instead of setting sprite anchor every time. As i said earlier, its better to assign in constructor, so its not important whether its null or `new Point` here.

Reply:
The feature is good even without TexturePacker. Setting texture default anchor instead of setting sprite anchor every time. As i said earlier, its better to assign in constructor, so its not important whether its null or new Point here.

Reply:
We have told many devs here that there are no new API features in v4. Supporting external tools like TexturePacker seems 🆗 though, however, Texture anchor should probably be v5 as a public API.